src/FixedDataTableCell.js

```diff
@@ -91,7 +91,7 @@ class FixedDataTableCell extends React.Component {
   }
 
   shouldComponentUpdate(nextProps) {
-    if (nextProps.isScrolling && this.props.rowIndex === nextProps.rowIndex) {
+    if ((nextProps.isScrolling || this.props.isScrolling) && this.props.rowIndex === nextProps.rowIndex) {
```

Comment:
Wouldn't we miss the update with `props.isScrolling` turned on to `false`, and the cell will never know about it.
I think some folks require to know about this flag (see recent [PR comment](https://github.com/schrodinger/fixed-data-table-2/pull/398#discussion_r261081942)).

Also is it really worth skipping a single update for this?

Reply:
Updates are missed even during scrolling, I feel we should group what prop we care about and then check them too along with isScrolling check.

Without this change every cell present in view port will be rendered once per scroll stop. If I have large number of rows and columns (100 rows and 10 columns ), and I quickly scroll multiple times (3 times) effectively 100 * 10 * 3 cells will be rendered.

Once migration to react 16 is done, can use context to provide scroll info to cells.

Reply:
> Updates are missed even during scrolling

Yea, that's the current optimization. With your change updates are missed even right after scrolling.

Let's say that scrolling is stopped, then on the next render shouldComponentUpdate is called, but this.props.isScrolling is still true. This will mean that the cells don't get updated. This will save a render but it's a required render as the cells need to know when scrolling has stopped.

If the cells don't know when scrolling has stopped, it will lead to problems. 
It breaks the API (see [comment](https://github.com/schrodinger/fixed-data-table-2/pull/398#discussion_r261081942)) and other prop changes can be lost until a new re-render happens.

> Without this change every cell present in view port will be rendered once per scroll stop. If I have large number of rows and columns (100 rows and 10 columns ), and I quickly scroll multiple times (3 times) effectively 100 * 10 * 3 cells will be rendered.

Not sure if I understand this, but If you scroll quickly, I would expect the existing shouldComponenttUpdate to skip render most of times. Or do you mean that you wait before each scroll? If that's the case, we still need to rerender because of the explanation above.

> Once migration to react 16 is done, can use context to provide scroll info to cells.

@wcjordan, we probably might need a new Issue/PR for this.